lib/documents/schemas/oim_projects.json

```diff
@@ -0,0 +1,114 @@
+{
+  "content_id": "4e4e33ff-271a-468a-b7a8-92d25f3f8ac0",
+  "base_path": "/oim-projects",
+  "format_name": "Office for the Internal Market project",
+  "name": "Office for the Internal Market projects",
+  "description": "Find reports and updates on current and historical OIM projects",
+  "filter": {
+    "format": "oim_case"
```

Comment:
Should this be `oim_project`?

Reply:
Yes! I just spotted it in the email signup [content item](https://www.integration.publishing.service.gov.uk/api/content/oim-projects/email-signup) I've pushed a fix, thank you!